src/Command/CommandBase.php

```diff
@@ -1155,4 +1155,98 @@ protected function convertEnvironmentAliasToUuid(InputInterface $input): void {
     }
   }
 
+  /**
+   * @param $cloud_environment
+   *
+   * @return string
+   */
+  public static function getSiteGroupFromSshUrl($cloud_environment): string {
+    $ssh_url_parts = explode('.', $cloud_environment->sshUrl);
+    $sitegroup = reset($ssh_url_parts);
+
+    return $sitegroup;
+  }
+
+  /**
+   * @param $cloud_environment
+   *
+   * @return bool
+   */
+  protected function isAcsfEnv($cloud_environment): bool {
+    if (strpos($cloud_environment->sshUrl, 'enterprise-g1') !== FALSE) {
+      return TRUE;
+    }
+    foreach ($cloud_environment->domains as $domain) {
+      if (strpos($domain, 'acsitefactory') !== FALSE) {
+        return TRUE;
+      }
+    }
+
+    return FALSE;
+  }
+
+  /**
+   * @param \AcquiaCloudApi\Response\EnvironmentResponse $cloud_environment
+   *
+   * @return array
+   * @throws \Acquia\Cli\Exception\AcquiaCliException
+   */
+  protected function getAcsfSites($cloud_environment): array {
+    $sitegroup = self::getSiteGroupFromSshUrl($cloud_environment);
+    $command = ['cat', "/var/www/site-php/$sitegroup.{$cloud_environment->name}/multisite-config.json"];
+    $process = $this->sshHelper->executeCommand($cloud_environment, $command, FALSE);
+    if ($process->isSuccessful()) {
+      return json_decode($process->getOutput(), TRUE);
+    }
+    throw new AcquiaCliException("Could not get ACSF sites");
+  }
+
+  /**
+   * @param \AcquiaCloudApi\Response\EnvironmentResponse $cloud_environment
+   *
+   * @return array
+   * @throws \Acquia\Cli\Exception\AcquiaCliException
+   */
+  protected function getCloudSites($cloud_environment): array {
+    $sitegroup = self::getSiteGroupFromSshUrl($cloud_environment);
+    $command = ['cat', "/var/www/site-php/$sitegroup.{$cloud_environment->name}/config.json"];
```

Comment:
I had no idea this file existed. How does it get populated?

Reply:
`config.json` is not a valid path on Cloud IDE. The only path that exists ATM is `/var/www/site-php/SITEGROUP/DBNAME-settings.php`

Reply:
`config.json` is part of the standard set of server-side includes on Cloud. It's not officially documented but AFAICT it's standard for every application, just like `multisite-config.json` is for ACSF.

We're calling this function on a `$cloud_environment` so it's fine that this file doesn't exist in IDEs.

Reply:
@anavarre I don't think this is looking for that path on Cloud IDE. It's looking for that path on a Cloud Platform env.